docs/index.md

```diff
@@ -18,9 +18,9 @@ metadata access for data and AI assets.
 You can get Gravitino from the [GitHub release page](https://github.com/datastrato/gravitino/releases),
 or you can build Gravitino from source code, please see [How to build Gravitino](./how-to-build.md).
 
-Gravitino runs on both Linux and macOS, and requires Java 8. Gravitino trino-connector runs with
+Gravitino runs on both Linux and macOS platforms, and it requires the installation of Java 8, Java 11, or Java 17 to function effectively. Gravitino trino-connector runs with
```

Comment:
can you please remove this " to function effectively".

Reply:
ok

Reply:
done